### DISCUSSION TITLE
Invitation to join the Power Platform Open-Source Hub initiative

### DISCUSSION THREAD

**Discussion by rpothin** [microsoft/Power-Fx#2208] [Ideas]
Hi team,

I hope you are doing well.

I am reaching out to you on behalf of the [Power Platform Open-Source Hub](https://rpothin.github.io/PowerPlatform-OpenSource-Hub/) initiative.

The goal of this initiative is to create a centralized place that simplifies the process of contributing to open-source projects related to Power Platform and provides guidance and support to individuals passionate about making a difference in this space.

I believe that your project would be a great addition to this initiative and would love to have you join it.

If you are interested in joining, please check out the [self-onboarding](https://rpothin.github.io/PowerPlatform-OpenSource-Hub/docs/repository-onboarding/self-onboarding) documentation page on the website of the initiative for more information.

A few complementary points regarding this request:

- There are already 26 repositories from the Microsoft organization referenced
- Adding topics to the repositories will help for their discoverability

Thank you for your time and consideration.

Best regards,
Raphael

**Comment by anderson-joyle** [comment]
Added `powerfx` and `power-fx` topics to the repository.

**Reply by rpothin** [reply]
Thank you very much @anderson-joyle 👍🏼
I will track it and confirm tomorrow when you the repository will be referenced in the website 😊

**Reply by rpothin** [reply]
Hi @anderson-joyle and team,

I confirm you have successfully onboarded the Power Platform Open-Source Hub initiative - and at the top of the list based on the stars count! Congrats 🥳🎉🎊 and thank you for your trust and your support.

<img width="1040" alt="image" src="https://github.com/microsoft/Power-Fx/assets/23240245/6a5b1079-9891-44e3-98c0-98d10e1acd92">

Have a great day.
Raphael
